pages/help.py

```diff
@@ -1046,12 +1046,9 @@ def get_content(self):
         return text(md(_("""
 Many of the developers and users of Freenet are on the [IRC](
 https://en.wikipedia.org/wiki/IRC) channel #freenet on chat.freenode.net.
-
-[Chat with us][url_chat]
-
 """) + "\n\n" + """
-[url_chat]: #chatlink "chatlink"
-"""))
+<a href="#chatlink" title="chatlink" class="btn button-custom btn-custom-two">{}</a>
```

Comment:
Now that you're inserting raw HTML anyways, I guess it would be better to stop misusing the `title` attribute here and instead use `id`.